randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -108,8 +108,11 @@ Extra - room_id: [3, 6]
   > Door to Entrance Lobby
       Any of the following:
           Can Climb High Jump Wall
-          # Needs to check if DLR in case it becomes an open hatch, making this impossible
-          Wall Jump (Advanced) and Disabled Door Lock Rando
+          All of the following:
+              # Needs to check if DLR in case it becomes an open hatch, making this impossible
+              Disabled Door Lock Rando
+              # https://youtu.be/weMdKWzp31g
+              Wall Jump (Advanced)
```

Comment:
not too sure about this being advanced tbh, seems pretty easy to me.
perhaps change this to WJ intermediate + knowledge beginner?